adal/build.gradle

```diff
@@ -149,6 +149,44 @@ task jacocoTestReport(type: JacocoReport, dependsOn: 'testDebugUnitTest') {
     ])
 }
 
+task createPom  {
+    pom {
+        project {
+            groupId 'com.microsoft.aad'
+            artifactId 'adal'
+            version = project.version
+            packaging 'aar'
+            name 'adal'
+
+            description 'Azure active directory library for Android gives you the ability to add Windows Azure Active Directory authentication to your application with just a few lines of additional code. Using our ADAL SDKs you can quickly and easily extend your existing application to all the employees that use Windows Azure AD and Active Directory on-premises using Active Directory Federation Services, including Office365 customers.'
+            url 'https://github.com/MSOpenTech/azure-activedirectory-library-for-android'
+
+            developers {
+                developer {
+                    id 'microsoft'
+                    name 'Microsoft'
+                }
+            }
+
+            licenses {
+                license {
+                    name 'MIT License'
+                }
+            }
+            inceptionYear '2014'
+
+            properties {
+                branch 'master'
+                adalVersion = project.version
+            }
+
+            scm {
+                url "https://github.com/MSOpenTech/azure-activedirectory-library-for-android/tree/master"
```

Comment:
Let's take this opportunity to update this url and remove the redirect. It should now be: `https://github.com/AzureAD/azure-activedirectory-library-for-android`